frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+import { TracimComponent } from '../../tracimComponent.js'
+import { appContentFactory } from '../../appContentFactory.js'
+import { tinymceRemove } from '../../helper.js'
+import CardPopup from '../CardPopup/CardPopup.jsx'
+import IconButton from '../Button/IconButton.jsx'
+
+const wysiwygId = 'wysiwygTimelineCommentEdit'
+const wysiwygIdSelector = `#${wysiwygId}`
+
+const CreateToDoFromTextPopup = (props) => {
+  useEffect(() => {
+    return tinymceRemove(wysiwygIdSelector)
+  }, [])
+
+  return (
+    <CardPopup
+      customClass='createToDoFromTextPopup'
+      customColor={props.customColor}
+      onClose={props.onClickClose}
+      label={props.t('Transorm todos')}
```

Comment:
```suggestion
      label={props.t('Transform to tasks')}
```

Reply:
Or 'Create task from text' like the button

Reply:
Done in 9d50a2d 
I have chosen `Create task from text`